### PR TITLE
[Improve](be)check swap

### PR DESCRIPTION
**Pull request by zzzzzzzs**
# Proposed changes

Check swap

## Problem summary

Check if swap is turned off

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: bin/start_be.sh [hunk 1]
```diff
@@ -71,6 +71,10 @@ if [[ "$(uname -s)" != 'Darwin' ]]; then
     fi
 fi
 
+if [[ "$(swapon -s | wc -l)" -gt 1 ]]; then
+    echo "Please disable swap partitioning before installation."
```
Comment (dataroaring) [inline bin/start_be.sh:75]: swap memory .

**Review by dataroaring — CHANGES_REQUESTED** [review]

File: bin/start_be.sh (hunk 1 above)
Comment (zzzzzzzs) [reply bin/start_be.sh]: OK

**Review by dataroaring — APPROVED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by yujun777 — APPROVED** [review]
LGTM

**Comment by hello-stephen** [comment]
run clickbench

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.02 seconds
 stream load tsv:          529 seconds loaded 74807831229 Bytes, about 134 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17161923744 Bytes

**Pull request merged**